content/blog/open-sourcing-workshops-on-demand-part-5-create-a-workshop.md

```diff
@@ -0,0 +1,236 @@
+---
+title: "Open sourcing Workshops-on-Demand - Part 5: Create a new workshop"
+date: 2025-09-26T08:24:03.351Z
+author: Frederic Passeron
+authorimage: /img/frederic-passeron-hpedev-192.jpg
+disable: false
+tags:
+  - Workshops-on-Demand
+  - Open Source
+---
+I﻿n this new article that is part of our series dedicated on [open sourcing of our Workshops-on-Demand project](https://developer.hpe.com/blog/willing-to-build-up-your-own-workshops-on-demand-infrastructure/), I will focus on the steps necessary  to build up a new workshop. I already covered most of the infrastructure parts that support the workshops. In my previous posts, I covered setting up the infrastructure to support the workshops. Now let's focus a little more on the content creation.
+
+# O﻿verview
+
+Let's start with a simple flowchart describing the 10000-foot view of the creation process:
+
+![](/img/wod-b-process.png "Workshop's creation flow.")
+
+A﻿s you can see, there's no rocket science here. Just common sense. Depending on the workshop you wish to create, some obvious requirements should show up. A workshop based on a programmatic language, for instance, may require the relevant kernel to be set up on the JupyterHub server. The following [page](https://gist.github.com/chronitis/682c4e0d9f663e85e3d87e97cd7d1624) lists all available kernels.
+
+S﻿ome other workshops might need a proper infrastructure to run on. A kubernetes101 workshop for instance could not exist without the presence of a proper Kubernetes cluster. The same thing goes for any HPE-related solutions.
+
+F﻿rom an infrastructure standpoint, a minimum number of environments are necessary. The project makes it easy to deploy a development and test environments, a staging environment, and at least one production environment. 
+
+Here are a few definitions of the different environments :
```

Comment:
Recommend rephrasing to:

"Let's look at how each of these environments is defined:"